### PR TITLE
Type override

### PR DESCRIPTION
**Pull request by sladyn98**
This PR is to test the nvm overridy type PR

### CODE REVIEW

**Comment by ljharb** [comment]
Looks like a single zsh test is still failing. However, it might be the test file itself and the way it's mocking things?

File: nvm.sh [hunk 1]
```diff
@@ -61,18 +61,18 @@ nvm_command_info() {
...
     # shellcheck disable=SC2230
-    INFO="$(which "${COMMAND}") ($(type "${COMMAND}" | command awk '{ $1=$2=$3=$4=$5="" ;print }' | command sed 's/^\ *//g'))"
-  elif type "${COMMAND}" | nvm_grep -q "^${COMMAND} is \\/"; then
-    INFO="$(type "${COMMAND}" | command awk '{print $3}')"
+    INFO="$(which "${COMMAND}") ($(command type "${COMMAND}" | command awk '{ $1=$2=$3=$4=$5="" ;print }' | command sed 's/^\ *//g'))"
+  elif command type "${COMMAND}" | nvm_grep -q "^${COMMAND} is \\/"; then
+    INFO="$(command type "${COMMAND}" | command awk '{print $3}')"
   else
-    INFO="$(type "${COMMAND}")"
+    INFO="$( type/ "${COMMAND}")"
```
Comment (ljharb) [inline nvm.sh]:
```suggestion
    INFO="$( \type "${COMMAND}")"
```

File: nvm.sh [hunk 2]
```diff
@@ -45,7 +45,7 @@ nvm_grep() {
 }
 
 nvm_has() {
-  type "${1-}" >/dev/null 2>&1
+  type "${1-}" >/dev/null 2>&1
```
Comment (ljharb) [inline nvm.sh]:
```suggestion
  \type "${1-}" >/dev/null 2>&1
```

File: nvm.sh [hunk 3]
```diff
@@ -61,18 +61,18 @@ nvm_command_info() {
...
     # shellcheck disable=SC2230
-    INFO="$(which "${COMMAND}") ($(type "${COMMAND}" | command awk '{ $1=$2=$3=$4=$5="" ;print }' | command sed 's/^\ *//g'))"
-  elif type "${COMMAND}" | nvm_grep -q "^${COMMAND} is \\/"; then
-    INFO="$(type "${COMMAND}" | command awk '{print $3}')"
+    INFO="$(which "${COMMAND}") ($(command type "${COMMAND}" | command awk '{ $1=$2=$3=$4=$5="" ;print }' | command sed 's/^\ *//g'))"
+  elif command type "${COMMAND}" | nvm_grep -q "^${COMMAND} is \\/"; then
+    INFO="$(command type "${COMMAND}" | command awk '{print $3}')"
   else
-    INFO="$(type "${COMMAND}")"
+    INFO="$( \type "${COMMAND}")"
```
Comment (ljharb) [inline nvm.sh]:
```suggestion
    INFO="$(\type "${COMMAND}")"
```

**Comment by sladyn98** [comment]
For this one we are getting something like this:
 `wget command info wrong(stage 1), expected: '/usr/bin/wget', got 'wget is /usr/bin/wget'`
 Isn't the grep command getting the correct the value since we are getting it now

**Comment by ljharb** [comment]
that seems right to me; can you update the test?

**Pull request closed without merging**